app/bundles/PluginBundle/Form/Type/IntegrationsListType.php

```diff
@@ -99,7 +99,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'attr'  => [
                         'class' => 'integration-config-container',
                     ],
-                    'integration' => (isset($integrationObjects[$data['integration']])) ? $integrationObjects[$data['integration']] : null,
+                    'integration' => (isset($integrationObjects[$data['integration'] ?? null])) ? $integrationObjects[$data['integration']] : null,
```

Comment:
This change doesn't make any sense. Either use `isset` or `??`. Why both?

Reply:
?? is on `$data['integration']` isset is on all `$integrationObjects[$data['integration'] ?? null]`

Two different variable. Just notice show up during the working on this PR

Reply:
Missed that it's 2 different arrays. Thanks for the explanation!